### ISSUE TITLE
Old bots no longer connect to remote config

### ISSUE THREAD

**Issue by briancodes799** [UnamSanctam/UnamWebPanel#412]
Unam there seems to be a critical bug with version `--cinit-version=3.1.0` I lost all my old bots because of this. None of them can't connect to the remote config that's left. For instance I had three configs like this `https://pastebin.com/raw/SKA2VVa3,https://rentry.co/miner1,https://rentry.co/miner2` the only link thats still up is the
one in the middle and there's no way I can get them to connect to it. Also `stealth-targets` does not seem to work also as I still have the old builds on my VM because I tried running it with 
`task manager` and `process hacker` open and it did not pause, it just continued mining regardless.I hope you can do something about this. Unfortunatley I dont have the miner checker or the uninstaller.

**Comment by UnamSanctam** [comment]
In the old version 3.1.0 then "Remote Configuration" backup links should work fine, are you sure you used the "raw" version of your rentry link? And for "stealth-targets" are you sure that you entered the correct .exe names for them? Also, the latest version 3.4.1 can be found here at the moment: https://github.com/UnamSanctam/UnamWebPanel/issues/361#issuecomment-2208790180

**Comment by briancodes799** [comment]
Yes I use the raw version for the `rentry` links I realized I didn't put in the the example I showed you. But I am just hoping that they will come back. I still use the old version because the new one uses the `VS` backend and I use `MinGW` to build it, it somehow has fewer detections for me since I use another version of `MinGW`.

**Comment by UnamSanctam** [comment]
Hmm weird, are you sure that the rentry configuration is valid JSON (you can check with a JSON validator)? And all versions of the miner should use MinGW, none of them use a VS backend. Using a different compiler version is always good (since the compiler patterns get detected) but you should be able to replace the MinGW in the latest version with your own as well.
